vuepress/docs/docs/README.md

```diff
@@ -37,16 +37,16 @@ Creating a GraphQL API with Caliban is as simple as creating a case class. Indee
 
 Let's say we have a class `Character` and 2 functions: `getCharacters` and `getCharacter`:
 
-```scala
+```scala mdoc:silent
 case class Character(name: String, age: Int)
 
-def getCharacters: List[Character] = ???
+def getCharacters: List[Character] = Nil
```

Comment:
Why can't this one be `???` and the next line can? This compiles, right?

Reply:
Yes, unfortunately. 

mdoc compiles and **evaluates** the snippet. 

There is a modifier [compile-only](https://scalameta.org/mdoc/docs/modifiers.html#compile-only), but the methods/variables will not be available in the sections below.